src/release/breaking-changes/trackpad-gestures.md

```diff
@@ -414,7 +414,8 @@ class Foo extends StatelessWidget {
 
 #### Using `RawGestureRecognizer`
 
-Explicitly ensure that `supportedDevices` does not include `PointerDeviceKind.trackpad`.
+Explicitly ensure that `supportedDevices`
+doesn't include `PointerDeviceKind.trackpad`.
```

Comment:
**question:** Why does this need to be `explicitly`? Can the developer add the `.trackpad` option without Flutter returning an error?